src/scene/renderer/renderer.js

```diff
@@ -56,10 +57,32 @@ const _fixProjRangeMat = new Mat4().set([
     0, 0, 0.5, 1
 ]);
 
+// helton sequence of 2d offsets for jittering
+const _haltonSequence = [
+    new Vec2(0.5, 0.333333),
+    new Vec2(0.25, 0.666667),
+    new Vec2(0.75, 0.111111),
+    new Vec2(0.125, 0.444444),
+    new Vec2(0.625, 0.777778),
+    new Vec2(0.375, 0.222222),
+    new Vec2(0.875, 0.555556),
+    new Vec2(0.0625, 0.888889),
+    new Vec2(0.5625, 0.037037),
+    new Vec2(0.3125, 0.370370),
+    new Vec2(0.8125, 0.703704),
+    new Vec2(0.1875, 0.148148),
+    new Vec2(0.6875, 0.481481),
+    new Vec2(0.4375, 0.814815),
+    new Vec2(0.9375, 0.259259),
+    new Vec2(0.03125, 0.592593)
+];
```

Comment:
Why bother creating `Vec2` instances? Won't an array of arrays be sufficient here?

Reply:
I don't know, I guess performance difference is minimal, it's a vector so I created a vector, but it could have been an array of just object too.